src/org/labkey/test/components/ui/navigation/UserMenu.java

```diff
@@ -45,4 +45,14 @@ protected Locator getToggleLocator()
     {
         return Locator.id(MENU_ID);
     }
+
+    /**
+     * The button for the drop down user menu. The user menu on the SampleManager app is an example of this button.
+     *
+     * @return A locator to the drop down user menu button.
+     */
+    public static Locator AppUserMenu()
+    {
+        return Locator.tagWithId("a", "user-menu-dropdown");
```

Comment:
Use the `MENU_ID` defined in this class.
```suggestion
        return Locator.tagWithId("a", MENU_ID);
```

Also, have `getToggleLocator` call and return this method.

Reply:
Updated.